contracts/dao/src/helpers.rs

```diff
@@ -61,14 +61,14 @@ pub fn get_voting_power_at_height(
     height: u64,
 ) -> StdResult<Uint128> {
     // Get voting power at height
-    let balance: ion_stake::msg::StakedBalanceAtHeightResponse = querier.query_wasm_smart(
+    let balance: ion_stake::msg::VotingPowerAtHeightResponse = querier.query_wasm_smart(
         staking_contract,
-        &ion_stake::msg::QueryMsg::StakedBalanceAtHeight {
+        &ion_stake::msg::QueryMsg::VotingPowerAtHeight {
             address: address.to_string(),
```

Comment:
```suggestion
            address: address.into_string(),
```

`to_string` does a heap allocation whereas `into_string` will just reuse the allocated memory if possible.